docs/guides/admin/docs/upgrade.md

```diff
@@ -84,7 +84,8 @@ But old publications still reference the players directly in the admin interface
 pose any immediate problem unless you actually want to switch players in which case you would need to republish the
 material once to update the publication links.
 
-Alternatively, you can rewrite the old links all at once without re-publication using the following method:
+Alternatively, you can rewrite the old links all at once without re-publication using the following method:  
+*Hint: replace the '...' with the actual path to your installation-storage*
```

Comment:
I think it's kind of self-explanatory, but in any case I'd prefer "Alternatively, you can rewrite the old links all at once without re-publication using the following method ( replace the '...' with the actual path to your installation):"

Reply:
#self-explanatory hmm it wasn't for me 
'..' is the previous folder, so i thought maybe this is some sed speciality i missed. Took me a while to figure that out - as those three dots look like real code